profiler/cppcheck-suppressions.txt

```diff
@@ -3,4 +3,6 @@
 *:*/test/*
 *:*/_deps/*
 // This error is an issue with cppcheck. So do not report this one specifically
-preprocessorErrorDirective:*/logging.h
\ No newline at end of file
+preprocessorErrorDirective:*/logging.h
+cppcheckError:*/StackSamplerLoopManager.cpp
+missingIncludeSystem:*
```

Comment:
remove missing standard include headers (too noisy). CppCheck folks does not recommend otherwise we might have events from those files (which is not what we want)